index.html

```diff
@@ -359,13 +353,7 @@ <h2>Glossary</h2>
 
-<p><dfn id="def_taskkil" class="lint-ignore export">Tashkil</dfn>. A tashkil (تَشْكِيل) is an Arabic script mark that indicates vocalization of text, as well as other types of phonetic guides to correct pronunciation. &#x62d;&#x654; [<span class="uname" translate="no">U+062D ARABIC LETTER HAH + U+0654 ARABIC HAMZA ABOVE</span>] is an example of a letter plus tashkil combination. Tashkil are separately encoded as combining marks. These include several subtypes: <em>harakat</em> (short vowel marks), <em>tanwin</em> (postnasalized or long vowel marks), <em>shaddah</em> (consonant gemination mark), and <em>sukun</em> (to mark lack of a following vowel). A basic Arabic letter plus any of these types of marks is never encoded as a separate, precomposed character, but must always be represented as a sequence of letter plus combining mark. Additional marks invented to indicate non-Arabic vowels, used in extensions of the Arabic script for languages other than Arabic, are also encoded as separate combining marks.</p>
-
-<aside class="note">
-	<p>This distinction between using a character with <a>ijam</a> instead of combining a letter with a <a>tashkil</a> becomes important when choosing which Unicode characters to use because (as can be seen in the examples above) the visual forms can be identical. Using the wrong character can change the meaning of the text, affecting the results of text search, font rendering, text to speech, etc.</p>
-
-    <p>There are, however, some very common combinations of diacritic and base that can be represented using precomposed characters or decomposed sequences that are canonically equivalent. For those the standard encourages the use of the precomposed form, but the fact that the forms are canonically equivalent removes concerns about changes in meaning.</p>
-</aside>
+<p><dfn id="def_tashkil" class="lint-ignore export">Tashkil</dfn>. A tashkil (تَشْكِيل) is an Arabic script mark that indicates vocalization of text, as well as other types of phonetic guides to correct pronunciation. &#x62d;&#x654; [<span class="uname" translate="no">U+062D ARABIC LETTER HAH + U+0654 ARABIC HAMZA ABOVE</span>] is an example of a letter plus tashkil combination. Tashkil are separately encoded as combining marks. These include several subtypes: <em>harakat</em> (short vowel marks), <em>tanwin</em> (postnasalized or long vowel marks), <em>shaddah</em> (consonant gemination mark), and <em>sukun</em> (to mark lack of a following vowel). A basic Arabic letter plus any of these types of marks is never encoded as a separate, precomposed character, but must always be represented as a sequence of letter plus combining mark. Additional marks invented to indicate non-Arabic vowels, used in extensions of the Arabic script for languages other than Arabic, are also encoded as separate combining marks. See also: <a>ijam</a>. For more information see <a href="https://r12a.github.io/scripts/arabic/arb.html#ijam_tashkil">this article</a>.</p>
```

Comment:
the arabic text needs markup and a lang tag

we should probably also add the arabic spelling for ijam if we have it here